src/python/pants/version.py

```diff
@@ -11,8 +11,10 @@
 _PANTS_VERSION_OVERRIDE = '_PANTS_VERSION_OVERRIDE'
 
 
-VERSION = (os.environ.get(_PANTS_VERSION_OVERRIDE) or
-           pkgutil.get_data(__name__, 'VERSION').strip().decode())
+VERSION: str = (
+  os.environ.get(_PANTS_VERSION_OVERRIDE) or
+  pkgutil.get_data(__name__, 'VERSION').decode().strip()  # type: ignore
```

Comment:
Nice catch on the ordering here!

Reply:
I don't think it makes a difference..but it seeems more correct to first convert to unicode, then deal with things like newlines.